src/components/posts/PostList.js

```diff
@@ -45,18 +48,25 @@ export const PostList = props => {
                 <th>Date</th>
                 <th>Category</th>
                 <th>Tags</th>
+                { isAdmin && <th>Approved</th> }
```

Comment:
It might be nice to add comments to explaing the intent anywhere we are using the && syntax to conditionally render components. This is becoming more familiar to me, but there are people in our cohort who wouldn't understand what this means.

Reply:
Suggested comment: `// if the user is an admin, render the Approved table header`